spec/misc.spec.js

```diff
@@ -0,0 +1,25 @@
+const assert = require('assert');
+const { exec, execSync } = require("child_process");
+
+describe('Run misc neu commands', () => {
+    describe('Test neu --help command', () => {
+        it('returns output of neu --help', (done) => {
+            exec("neu --help", (error, stdout, stderr) => {
+                if(error) {
+                    done(error);
+                }
+                else {
+                    if(stderr) {
+                        done(stderr);
+                    }
+                    assert.equal(error, null);
+                    assert.equal(stderr, '');
+                    assert.ok(typeof stdout == 'string');
+                    assert.ok(stdout.includes('Usage: neu [options] [command]'));
+                    assert.ok(stdout.includes(''));
+                    done();
+                }
+            }).on('exit', (status) => { assert.strictEqual(status, 0); });
```

Comment:
This is only used once right now, but later, we have to do the same thing in other files as well. Therefore, could you please put this code in `runner.js` similar to the Neutralinojs main repo spec? I think we can create an async function thereby wrapping the `exec` function. Also, I think we don't need to subscribe to the `exit` event here if we use `execSync` with a try-catch block

Thanks :tada:

Reply:
The first part, yea, cool. Makes sense. I will look into `runner.js` and work on that. For `exit` event, do you mean to wrap the function in a try-catch block and throw an error via try-catch block, instead of checking for the status code?